ext/readline/tests/libedit_info_001.phpt

```diff
@@ -24,13 +24,13 @@ var_dump(readline_info('attempted_completion_over'));
 
 ?>
 --EXPECTF--
-array(6) {
+array(%d) {
   ["line_buffer"]=>
   string(0) ""
   ["point"]=>
   int(0)
   ["end"]=>
-  int(0)
+  int%a
```

Comment:
question: is the optional element in the next line ?

Reply:
Yes, this is the output when using the --with-libedit:

```
array(7) {
  ["line_buffer"]=>
  string(0) ""
  ["point"]=>
  int(0)
  ["end"]=>
  int(0)
  ["erase_empty_line"]=>
  int(0)
  ["library_version"]=>
  string(16) "EditLine wrapper"
  ["readline_name"]=>
  string(0) ""
  ["attempted_completion_over"]=>
  int(0)
}
```

and this is the output when using --with-readline:

```
array(14) {
  ["line_buffer"]=>
  string(0) ""
  ["point"]=>
  int(0)
  ["end"]=>
  int(0)
  ["mark"]=>
  int(0)
  ["done"]=>
  int(0)
  ["pending_input"]=>
  int(0)
  ["prompt"]=>
  string(0) ""
  ["terminal_name"]=>
  string(0) ""
  ["completion_append_character"]=>
  string(1) " "
  ["completion_suppress_append"]=>
  bool(false)
  ["erase_empty_line"]=>
  int(0)
  ["library_version"]=>
  string(3) "8.2"
  ["readline_name"]=>
  string(5) "other"
  ["attempted_completion_over"]=>
  int(0)
}
```

Reply:
Ok. Ideally, I wish we do not lose the int(0) information.

Reply:
Changed this to `%A` tag - to match zero or more of anything including newlines. 
https://qa.php.net/phpt_details.php#expectf_section

Yes, this now is a bit better.